src/app/docs/button/toolbar-guidelines.component.ts

```diff
@@ -18,14 +18,32 @@ public addButtonInfo: {label: string, value: string}[] = [
   },
   {
     label: 'Dropdown',
-    value: 'Optional. Allows users to add more than one type of record.'
+    value: 'Optional. Allows users to add more than one type of item.'
   },
   {
     label: 'Notes',
     value: 'If add is the primary action, use the blue primary button styling. Otherwise, use the white secondary button styling.'
   }
 ];
-public editButtonInfo: {label: string, value: string}[] = [
+public newButtonInfo: {label: string, value: string}[] = [
+  {
+    label: 'Text',
+    value: 'New'
+  },
+  {
+    label: 'Icon',
+    value: 'sky-i-add'
```

Comment:
The icon listed here and the modern icons that follow don't match up with what I see at https://developer.blackbaud.com/skyux-indicators/docs/icon?svcid=skyux-modern#section-custom-icons-beta. I assume that what you have in place is correct, but I want to check to make sure.

Reply:
yes. In that modern icon table, it should label the icon as "New" instead of "Add"

Reply:
OK. Pendleton has a separate PR for icons. I'll get back to that today and make sure we are consistent.